samples/drivers/aio_comparator/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+    description: TBD
+    name: TBD
+tests:
+-   test:
+        arch_whitelist: x86
+        build_only: true
```

Comment:
suggest loosing this build_only and make it a tag (eg: tags: build_only, slow, etc) instead of adding fields